Common/UnitDefinitions/Pressure.json

```diff
@@ -321,6 +321,30 @@
           "Abbreviations": [ "inHg" ]
         }
       ]
+    },
+    {
+      "SingularName": "DynePerSquareCentimeter",
+      "PluralName": "DynesPerSquareCentimeter",
+      "FromUnitToBaseFunc": "x*1.0e-1",
+      "FromBaseToUnitFunc": "x/1.0e-1",
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "dyn/cm²" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "PoundPerInchSecondSquared",
+      "PluralName": "PoundsPerInchSecondSquared",
```

Comment:
Is this a pressure unit? `Second` seems out of place to me and I can't Google it in context of pressure.

Reply:
Yes, see [here](https://www.wolframalpha.com/input/?i=lbm%2F(in+sec%5E2)).

You can also see it corresponds to the base dimensions: mass / (length * time²)

```json
  "BaseDimensions": {
    "L": -1,
    "M": 1,
    "T": -2
  }
```

Reply:
Thanks

Reply:
👍